app/graphiql.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path')
+
+module.exports = (req, res, next) => {
+  if (req.query.query) {
```

Comment:
Is the query parameter sufficient or should we also check the accept header?

Reply:
Query parameter should be sufficient. Checking the Accept header risks not being able to test this from the browser, depending on how strict the check is.

Reply:
fair point. I'm convinced :)

Reply:
I believe GraphiQL 1 had a permalink feature where the `query` parameter would be used to prefill the query textfield in the UI. However, as far as I can tell, this feature does not seem to exist in GraphiQL 2, so there currently does not seem to be a need to redirect some requests to GraphiQL.